cl/users/templates/profile/delete.html

```diff
@@ -16,7 +16,7 @@ <h3>This <em>cannot</em> be undone.</h3>
       will be deleted and resurrecting your account will not be
       possible.
   </p>
-  {% if request.user.alerts.count or request.user.favorites.count or non_deleted_map_count %}
+  {% if request.user.alerts.count or request.user.favorites.count or non_deleted_map_count or request.user.user_tags.count%}
```

Comment:
```suggestion
  {% if request.user.alerts.count or request.user.favorites.count or non_deleted_map_count or request.user.user_tags.count or request.user.docket_alerts.count%}
```

Since the method `delete_user_assets` also deletes DocketAlerts, would be useful to add it here?

Reply:
Yep. Good point.